Documentation/helm-ceph-cluster.md

```diff
@@ -32,20 +32,68 @@ Rook currently publishes builds of this chart to the `release` and `master` chan
 
 The release channel is the most recent release of Rook that is considered stable for the community.
 
+**Before installing, review the values.yaml to confirm if the default settings need to be updated.
+The [defaults](https://github.com/rook/rook/tree/{{ branchName }}/cluster/charts/rook-ceph-cluster/values.yaml)
+are only an example and not likely to apply to your cluster.**
+
+The example install assumes you have created a values-override.yaml.
+
 ```console
 helm repo add rook-release https://charts.rook.io/release
-helm install --create-namespace --namespace rook-ceph rook-ceph --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster
+helm install --create-namespace --namespace rook-ceph rook-ceph-cluster \
+   --set operatorNamespace=rook-ceph rook-release/rook-ceph-cluster -f values-override.yaml
+```
+
+## Configuration
```

Comment:
Some of these sections were just reordered for the common install case to be first.